apis/Google.Cloud.Dialogflow.V2/docs/history.md

```diff
@@ -1,5 +1,18 @@
 # Version history
 
+# Version 3.0.0-beta01, released 2020-04-06
+
+- [Commit 5a41fb5](https://github.com/googleapis/google-cloud-dotnet/commit/5a41fb5):
+  - Change `parent` field's resource_reference to specify child_type instead of type per client library generation requirement;
+  - Change Session with its child resource pattern to support both projects/{project}/agent/sessions/{session} and projects/{project}/agent/environments/{environment}/users/{user}/sessions/{session};
```

Comment:
And remove the one requiering location?

Reply:
This was just what's already in the commit, and the Dialogflow team didn't include that bit. I'll add it now, just in the update history.